slapr/slack.py

```diff
@@ -6,7 +6,8 @@
 import re
 from typing import List, NamedTuple, Optional, Set
 
-import slack
+import slack_sdk
+import slack_sdk.errors
```

Comment:
```suggestion

```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do you need this one?